docs/guides/windows_service.md

```diff
@@ -3,71 +3,76 @@ layout: default
+
 ```
 node -v
 ```
 
 The output should be:
-> PS C:\Users\SomeUser> node -v
-<br/> v18.14.2
-
+```
+PS C:\Users\SomeUser> node -v
+v18.14.2
+```
+---
 ### Step 2) Installing MongoDB 5+
-Install MongoDB Community Edition. This URL should allow you to download the MongoDB installation setup: 
-[MongoDB Download](https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-6.0.5-signed.msi). 
-If not visit [MongoDB MSI download page](https://www.mongodb.com/try/download/community).
 
-### Step 2b) (Optional)
-Please skip this if you're not an experienced user! 
+Install MongoDB Community Edition from [https://www.mongodb.com/try/download/community](https://www.mongodb.com/try/download/community). You can use [this URL](https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-6.0.5-signed.msi) to download the MongoDB installation setup if the previous link doesn't work.
 
-MongoDB provides extra tools for you to get insight in your database, the [MongoDB Developer Tools](https://www.mongodb.com/developer-tools).
+---
+### Step 2b) (Optional)
 
-The following tools might be of interest:
-- [Compass](https://www.mongodb.com/products/compass) connects to your database and query/adjust collections or documents.
-- [MongoDB VS Code Extension](https://www.mongodb.com/products/vs-code), inside VS Code, it connects to your database and see/adjust the data in place.
-- [MongoDB shell](https://www.mongodb.com/products/shell) provides shell access to the database. For advanced users only!
+If you're an experienced user, you might want to install the [MongoDB Developer Tools](https://www.mongodb.com/developer-tools), which provide extra tools to get insight into your database. The following tools might be of interest:
 
+- [Compass](https://www.mongodb.com/products/compass): Connects to your database and query/adjust collections or documents.
+- [MongoDB VS Code Extension](https://www.mongodb.com/products/vs-code): Connects to your database inside VS Code and allows you to see/adjust data in place.
+- [MongoDB Shell](https://www.mongodb.com/products/shell): Provides shell access to the database. For advanced users only!
+---
 ### Step 3) Installing Git
-Prepare the installation, by ensuring you have git installed. This will help you in updating FDM Monster in the future.
-Find it here: [Git Download](https://git-scm.com/downloads)
 
-### Step 4)
-The following steps 5-8 will install FDM Monster as a service manually. If you'd like to skip this, please use the `installations/fdm-monster-node-windows` where I've provided scripts that will install the necessary stuff for you.
+Prepare the installation by ensuring you have Git installed. This will help you in updating FDM Monster in the future. Find it here: [Git Download](https://git-scm.com/downloads)
 
+
+---
+
+### Step 4)
 The first time I would run the powershell script `download-fdm-monster-server.ps1` inside a powershell instance:
```

Comment:
`During first installation, it's required to run the powershell script download-fdm-monster-server.ps1 inside a powershell instance:`